core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -13,19 +13,13 @@
 // limitations under the License.
 
 /**
- * @fileoverview Tests for CollectionNodeObjectFactory.
+ * @fileoverview Tests for collection-node.model.
```

Comment:
```suggestion
 * @fileoverview Tests for CollectionNodeModel.
```

So that it matches the pattern used in other files.